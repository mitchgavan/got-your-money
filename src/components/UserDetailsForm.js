import React from 'react';

export default () => (
  <form>
    <label>
      <span>Weekly income</span>
      <input type="text" />
    </label>
  </form>
);
