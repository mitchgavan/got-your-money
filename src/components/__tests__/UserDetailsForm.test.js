import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import UserDetailsForm from '../UserDetailsForm';

test('It renders correctly', () => {
  const tree = shallow(
    <UserDetailsForm />,
  );
  expect(toJson(tree)).toMatchSnapshot();
});
