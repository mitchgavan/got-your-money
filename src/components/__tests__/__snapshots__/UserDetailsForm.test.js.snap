// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders correctly 1`] = `
<form>
  <label>
    <span>
      Weekly income
    </span>
    <input
      type="text"
    />
  </label>
</form>
`;
